modules/custom/az_carousel/az_carousel.module

```diff
@@ -0,0 +1,15 @@
+<?php
+
+/**
+ * @file
+ * Contains az_carousel.module.
+ */
+
+/**
+ * Implements hook_preprocess_node().
+ */
+function az_carousel_preprocess_node(&$variables) {
```

Comment:
It looks like this may be adding the carousel library as a dependency to every node.

Reply:
Agreed.  It'd be good to scope this to just the one content type.